README.md

```diff
@@ -59,6 +59,9 @@ Contributions, issues, and feature requests are welcome!
 
 Give a ⭐ if you like this project!
 
+## Acknowledgement
+Acknowledgments Great appreciation to Microverse community. Much appreciate to Nelson Sakwa Createor of design 📝 License This project is MIT licensed.
+
```

Comment:
- [ ]  Well done for acknowledging Nelson Sakwa. I recommend you link back to the [original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) so anyone can easily see where you got your design inspiration thereby giving the author appropriate credit. This is the essence of the acknowledgement 😎